torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -122,6 +122,33 @@ def powerSGD_hook(
     input_tensor = bucket.get_tensors()[0]
     device = input_tensor.device
     dtype = input_tensor.dtype
+
+    # Incorporate the error from the previous state into the gradients.
+    bucket_index = bucket.get_index()
+    input_tensor_cp = None
+    total_length = input_tensor.shape[0]
+    if state.use_error_feedback:
+        # The buckets can be rebuilt during training.
+        # In this case, the error tensor shape will not be aligned with the input tensor,
+        # and the error will be re-initialized as zeros.
+        if (
+            bucket_index in state.error_dict
+            and state.error_dict[bucket_index].shape[0] == total_length
+        ):
+            input_tensor.add_(state.error_dict[bucket_index])
+        else:
+            logging.info(
```

Comment:
Nit: can we bump this to a `logging.debug()`? It seems more useful for gradient compression developers rather than users.

Reply:
I think this log info is equally important as the one ["Reducer buckets have been rebuilt in this iteration"](https://github.com/pytorch/pytorch/blob/master/torch/nn/parallel/distributed.py#L672). Both of these logs appear very infrequently.